blog/templates/blog/admin/category_manage.html

```
@@ -0,0 +1,241 @@
+{% extends "blog/admin/base.html" %}
+{% load static %}
+
+{% block title %}카테고리 관리{% endblock %}
+
+{% block content %}
+<div class="layout">
+    <h1>카테고리 관리</h1>
+
+    <form method="post" action="{% url 'blog:admin-category-create' %}" class="filter-box" id="category-create-form">
+        {% csrf_token %}
+        <input type="text" name="name" placeholder="카테고리 이름 입력" maxlength="20" required>
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)